models/balancer/arbitrum/balancer_v2_arbitrum_protocol_revenue.sql

```diff
@@ -0,0 +1,20 @@
+{% set blockchain = 'arbitrum' %}
+
+{{
+    config(
+        schema='balancer_v2_arbitrum',
+        alias = 'protocol_revenue', 
+        materialized = 'table',
```

Comment:
I'm wondering if this macro/spell couldn't be incremental - probably Dune guys will ask the same